Console/Command/AclExtrasShell.php

```diff
@@ -21,7 +21,7 @@
  * @package		acl_extras
  * @subpackage	acl_extras.Console.Command
  */
-class AclExtrasShell extends Shell {
+class AclExtrasShell extends AppShell {
```

Comment:
Not all applications have an `AppShell` though.

Reply:
Ok, but every Shell in the CakePHP 2.x lib itself extends the AppShell, wouldn't that break stuff too then?

Reply:
To test this further I've removed the AppShell.php from my app/Console/Command directory and the AclExtras console commands still work fine. I expect this has to do with the AppShell in the lib being used.

Reply:
Oops I forgot this was for 2.x